docs/examples/stale-data.py

```diff
@@ -0,0 +1,92 @@
+"""
+Stale Data Periods
+===================
+
+Identifying stale data periods, defined as periods of
+consecutive repeating values, in time series.
+"""
+
+# %%
+# Identifing and removing stale, or consecutive repeating, values in time
+# series data reduces noise when performing data analysis. This example shows
+# how to use two PVAnalytics functions,
+# :py:func:`pvanalytics.features.gaps.stale_values_diff`
+# and :py:func:`pvanalytics.features.gaps.stale_values_round`, to identify
+# and mask stale data periods in time series data.
+
+import pvanalytics
+from pvanalytics.quality import gaps
+import matplotlib.pyplot as plt
+import pandas as pd
+import pathlib
+import numpy as np
+
+# %%
+# First, we import the AC power data stream that we are going to check for
+# stale data periods. The time series we download is a normalized AC power time
+# series from the PV Fleets Initiative, and is available via the DuraMAT
+# DataHub:
+# https://datahub.duramat.org/dataset/inverter-clipping-ml-training-set-real-data
+
+pvanalytics_dir = pathlib.Path(pvanalytics.__file__).parent
+file = pvanalytics_dir / 'data' / 'ac_power_inv_2173.csv'
+data = pd.read_csv(file, index_col=0, parse_dates=True)
+data = data.asfreq("15T")
+
+# %%
+# We plot the time series before inserting artificial stale data periods.
+data.plot()
+plt.xlabel("Date")
+plt.ylabel("Normalized AC Power")
+plt.legend(labels=["AC Power"])
+plt.tight_layout()
+plt.show()
+
+# %%
+# We insert some repeating/stale data periods into the time series for the
+# stale data functions to catch, and re-visualize the data, with those stale
+# periods masked
+
+data[460:520] = data.iloc[460]
+data[755:855] = data.iloc[755]
+data[1515:1600] = data.iloc[1515]
+stale_data_insert_mask = pd.Series([False] * len(data), index=data.index)
```

Comment:
What you have is more explicit, but FYI `pd.Series(False, index=data.index)` is equivalent

Reply:
Changed to this in 
commit 839c794a37982d4b12951cbb283d33774f27bb03